index.html

```diff
@@ -339,73 +340,62 @@ <h3>Methods</h3>
-          this amount of time has passed in seconds, allowing the <a>user agent</a>
-          to perform more or less buffering than it might otherwise do.
-          This allows to influence the tradeoffs between having a higher delay
-          and the risk that buffers such as the jitter buffer will run out of
-          audio or video frames to play due to network jitter.</p>
-          <p>The <a>user agent</a> may have a <dfn>minimum allowed delay</dfn> and a
-          <dfn>maximum allowed delay</dfn> reflecting what the <a>user agent</a> is
+          of time in milliseconds of media for the {{RTCRtpReceiver}}'s jitter
+          buffer to hold. This influences the amount of buffering done by the
+          <a>user agent</a>, and allows applications to control the tradeoff between
+          playout delay and the risk of running out of audio or video frames due to
+          network jitter.
+
+          <p>The <a>user agent</a> MUST have a <dfn>minimum allowed depth</dfn> and a
+          <dfn>maximum allowed depth</dfn> reflecting what the <a>user agent</a> is
           able or willing to provide based on network conditions and memory
           constraints.</p>
-          <p class="note">
-            The playout delay hint applies even if DTX is used. For example, if
-            DTX is used and packets start flowing after silence, the hint can
-            influence the <a>user agent</a> to buffer these packets rather than playing
-            them out.
-          </p>
-          <p class="note">
-            If the track is paired with other tracks through
-            {{RTCRtpReceiver}}
-            <a data-cite="WEBRTC#dfn-associatedremotemediastreams">
-            [[\AssociatedRemoteMediaStreams]]</a> internal slot, then it will
-            be synchronized with other tracks (for e.g. audio video
-            synchronization). This means that even if one of the paired tracks
-            is delayed through {{RTCRtpReceiver/[[PlayoutDelay]]}} then the <a>user agent</a>
-            synchronization mechanism will automatically delay all others
-            paired tracks. If multiple such paired tracks are delayed through
-            {{RTCRtpReceiver/[[PlayoutDelay]]}} by different amounts then the largest
-            of those hints will take precedence in synchronization mechanism.
-          </p>
-          <p class="note">
-            The receiver's average delay can be measured as the delta
+          <div class="note">
+            <p>This is a target value. The resulting change in delay can be gradually
+            observed over time. The receiver's average jitter buffer delay can be
+            measured as the delta
             {{RTCInboundRtpStreamStats/jitterBufferDelay}} divided by the delta
             {{RTCInboundRtpStreamStats/jitterBufferEmittedCount}}.
-          </p>
+            </p>
+            <p>
+            An average delay is expected even if DTX is used. For example, if
```

Comment:
What's DTX?

Reply:
It's [existing text](https://w3c.github.io/webrtc-extensions/#dom-rtcrtpreceiver-playoutdelay). I believe it's https://www.rfc-editor.org/rfc/rfc4788#section-1.3

Reply:
Ack

Reply:
https://www.rfc-editor.org/rfc/rfc7587#section-3.1.3 is a better reference.